templates/404/invalid_protected_area.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html' %}
+{% block content %}
+<h3>Invalid protected area</h3>
+<p>
+  The protected area ID you provided is not supported.
```

Comment:
In a more recent comment (for the version of this page that was moved to the `422` folder) you said "not supported" is okay because we're making arbitrary protected area IDs. I'll stick with "not supported" language for protected area IDs.